packages/wrangler/src/deployment-bundle/guess-worker-format.ts

```diff
@@ -38,6 +38,7 @@ export default async function guessWorkerFormat(
 		metafile: true,
 		bundle: false,
 		write: false,
+		logLevel: "silent",
```

Comment:
When guessing the worker format, we use esbuild but were forgetting to silence its stdout logging.
This led to duplicate logs and also confused things where we were relying on this run of esbuild to output initial build failures.

Reply:
Landed in a previous PR.